e2e/tests/stress_cpu.robot

```diff
@@ -12,65 +17,60 @@ Test Teardown    Cleanup test resources
 ${LOOP_COUNT}    1
 ${RETRY_COUNT}    300
 ${RETRY_INTERVAL}    1
-
 *** Test Cases ***
 
 Stress Volume Node CPU When Replica Is Rebuilding
-    Given Create a volume with 5 GB and 3 replicas
-    And Write data to the volume
+    Given Create volume 0 with 5 GB and 3 replicas
+    And Attach volume 0
+    And Write data to volume 0 endpoint
 
     FOR    ${i}    IN RANGE    ${LOOP_COUNT}
-        When Delete replica on volume node to trigger replica rebuilding
-        And Stress the CPU of all volume nodes
+        When Delete volume 0 replica on volume node
+        And Wait until volume 0 replica rebuilding started on volume node
+        And Stress the CPU of node with volume 0
 
-        Then Wait until replica on volume node rebuilt
-        And Check data is intact
+        Then Wait until volume 0 replica rebuilding completed on volume node
+        And Check volume 0 data is intact
     END
 
 Stress Volume Node CPU When Volume Is Detaching and Attaching
-    Given Create a volume with 5 GB and 3 replicas
-    And Write data to the volume
+    Given Create volume 0 with 5 GB and 3 replicas
+    And Attach volume 0
+    And Write data to volume 0 endpoint
 
     FOR    ${i}    IN RANGE    ${LOOP_COUNT}
-        When Stress the CPU of all volume nodes
-
-        And Detach volume from node
-        And Attach volume to node
+        When Stress the CPU of node with volume 0
+        And Detach volume 0
+        And Attach volume 0
 
-        And Check data is intact
+        Then Check volume 0 data is intact
     END
 
 Stress Volume Node CPU When Volume Is Online Expanding
-    @{data_checksum_list} =    Create List
-    Set Test Variable    ${data_checksum_list}
-
-    Given Create statefulset 0 with rwo volume
-    And Write 1024 MB data to statefulset 0
+    Given Create statefulset 0 using RWO volume
+    And Write 1024 MB data to file 0 in statefulset 0
 
     FOR    ${i}    IN RANGE    ${LOOP_COUNT}
-        And Stress the CPU of all volume nodes
-        When Expand statefulset 0 volume by 100 MiB
+        And Stress the CPU of all nodes with volume
+
+        When Expand statefulset 0 volume 0 by 100 MiB
 
-        Then Wait for statefulset 0 volume size expanded
-        And Check statefulset 0 data is intact
+        Then Wait for statefulset 0 volume 0 size expanded
```

Comment:
Why do we need to specify `statefulset 0 volume 0`? Are there multiple volumes?

Reply:
Statefulset is capable of having multiple volumes. However, since there are no current requirements for multiple volume testings, I've removed the index specification from the keywords.